assets/src/users/UserContext.tsx

```diff
@@ -48,16 +55,21 @@ const useAuthUser = () => {
   const { user } = useAuth0();
   return user;
 };
-const usePayUser = () => React.useContext(PayUserContext);
+const usePayUser = () => React.useContext(PayUserContext)!;
```

Comment:
@acodegirl this `!` makes me nervous. what led to this being required? I wonder if there's a way to not use this?